packages/backend/src/graphql/resolvers/incoming_payment.test.ts

```diff
@@ -68,4 +102,149 @@ describe('Incoming Payment Resolver', (): void => {
       }
     })
   })
+
+  describe('Mutation.createIncomingPayment', (): void => {
+    test.each`
+      description  | externalRef  | desc
+      ${'rent'}    | ${undefined} | ${'description'}
+      ${undefined} | ${'202201'}  | ${'externalRef'}
```

Comment:
You can also test with/without `incomingAmount` and `expiresAt`
(The returned created incoming payment will always include `expiresAt`)